packages/components/src/components/page-error/page-error.jsx

```diff
@@ -11,7 +12,9 @@ const PageError = ({
     image_url,
     messages,
     redirect_label,
+    redirect_labels,
     redirect_url,
+    redirect_urls,
```

Comment:
Is there a better way where we can combine, `redirect_urls` and `redirect_url` into a single prop and handle it accordingly?

Reply:
We can use one prop, but there are two ways:
- We can pass either Array or string and use condition here to check which type was passed
- We can always pass an array, but then we will need to change all the existing PageError calls to pass an array of a single element instead of a string
Which one do you think is the best way for that?

Reply:
I prefer second one as we are clear about the data type passing.

Reply:
Alright, changed accordingly